server/controllers/eventsController.ts

```diff
@@ -0,0 +1,137 @@
+import { Request, Response } from 'express';
+import { Event } from 'server/models/Event';
+
+export default {
+  async index(req: Request, res: Response) {
+    const { chapter_id } = req.params;
+
+    const events = await Event.find({ where: { chapter: chapter_id } });
+
+    res.json(events);
```

Comment:
You might not want to return the entire event

Reply:
This should be a general REST API, so here we should be returning the whole event, shouldn't we?

Reply:
yep but there could be sensitive info in the database row in the future, maybe just leave a comment?

Reply:
I'll comment in the model